src/slim-select/index.ts

```diff
@@ -462,6 +459,13 @@ export default class SlimSelect {
     // If there is no slim dont do anything
     if (!slim || !select) { return }
 
+
+    document.removeEventListener('click', this.functionNamespace.documentClick)
+
+    if (this.config.showContent === 'auto') {
+      window.removeEventListener('scroll', this.functionNamespace.windowScroll)
```

Comment:
note that third argument should match between add and remove

```suggestion
      window.removeEventListener('scroll', this.windowScroll, false)
```